lib/commands/condition.js

```diff
@@ -75,20 +81,29 @@ export default {
   },
 
   /**
-   * Enable condition inducer. The socket needs to remain connected during operation
-   * conditionID  Condition[0].identifier
-   * profileID    Condition[0].profiles[0].identifier
-   * @param { {conditionID: string, profileID: string} } options
-   * @returns {Promise<boolean>} `true` if enable the condition succeeded
-   * @throws {Error} If try to start another Condition and the previous Condition has not stopped, it will throw an error
+   * Enable a "condition inducer". You can create a condition on a connected device to test your app under adverse conditions,
+   * such as poor network connectivity or thermal constraints. When you start a device condition,
+   * the operating system on the device behaves as if its environment has changed. The device
+   * condition remains active until you stop the device condition or disconnect the device. For
+   * example, you can start a device condition, run your app, monitor your app’s energy usage, and
+   * then stop the condition.
+   *
+   * (Note: the socket needs to remain connected during operation)
+   * (Note: Device conditions are available only for real devices running iOS 13.0 and later.)
+   *
+   * @param {string} conditionID - Determine which condition IDs are available with the {@linkcode listConditionInducers} command
+   * @param {string} profileID - Determine which profile IDs are available with the {@linkcode listConditionInducers} command
+   * @returns {Promise<boolean>} `true` if enabling the condition succeeded
+   * @throws {Error} If you try to start another Condition and the previous Condition has not stopped
+   * @since 4.9.0
+   * @see {@link https://help.apple.com/xcode/mac/current/#/dev308429d42}
```

Comment:
I think this is redundant, but I'd have to look at the output to know for sure. as opposed to:

```js
/**
 * @see https://somewhere.com
 */
```